src/h3lib/lib/h3Index.c

```diff
@@ -293,7 +293,7 @@ int H3_EXPORT(compact)(const H3Index* h3Set, H3Index* compactedSet,
                 int loc = (int)(parent % numRemainingHexes);
                 int loopCount = 0;
                 while (hashSetArray[loc] != 0) {
-                    if (loopCount > numRemainingHexes) {
+                    if (loopCount > numRemainingHexes) {  // LCOV_EXCL_BR_LINE
                         // LCOV_EXCL_START
```

Comment:
Do we need this comment too?

Reply:
`LCOV_EXCL_BR_LINE` only excludes checking for branch coverage on the line it's on. Lcov doesn't have a concept of a set of lines being in a branch, so it doesn't exclude this block.